Pod/Classes/CDTQQuerySqlTranslator.m

```diff
@@ -358,19 +350,27 @@ + (CDTQSqlParts *)wherePartsForAndClause:(NSArray *)clause
                     [sqlParameters addObject:[negatedPredicate objectForKey:operator]];
 
             } else {
-                NSString *sqlOperator = notOperatorMap[operator];
-
-                if (!sqlOperator) {
-                    LogError(@"Unsupported comparison operator %@", operator);
-                    return nil;
+                NSString *sqlClause;
+                if ([operator isEqualToString:NE]) {
+                    // Treat $not..$ne as $eq
+                    NSString *sqlOperator = operatorMap[EQ];
+                    sqlClause = [NSString stringWithFormat:@"\"%@\" %@ ?", fieldName,
+                                   sqlOperator];
```

Comment:
This "treat blah as blah" feels like it should be dealt with during normalisation of the query rather than special cased down here?